### PR TITLE
Heroku Setup

### PR DESCRIPTION
**Pull request by aatifjiwani**
Preliminary app setup.
Created styling for the home#index page
Included footer templates within their own files and rendered them on index.html.erb

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+    def page_title
+        @pg_title ||= "Haxxor News"
+    end
```
Comment (dbernheisel) [inline app/controllers/application_controller.rb]: Looks like you're using 4 spaces for indentation. The [general Ruby style](https://github.com/rubocop-hq/ruby-style-guide) is to use 2 spaces. I'd recommend using something like rubocop or an editor plugin to help you learn what those styles are and enforce them in your code. At the very least, being consistent is more important than being a "4 spaces" or "2 spaces" person.

Normally I wouldn't care about style in code reviews, but some folks really care when reviewing your github history.

File: app/views/home/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+
+<header class="site-header" role="banner">
+  <div class="wrapper">
+      <p class="site-title" >Haxxor News</p>
+      <nav class="site-nav">
+        <div class="trigger">
+            <%= link_to 'Create Post', nil, {:class => 'page-link'} %>
+      </div>
+      </nav></div>
```
Comment (dbernheisel) [inline app/views/home/index.html.erb]: Indentation is off here. Best practice for readability is to have the opening tag and closing tag on the same indentation level.

ie
```html
<div class="wrapper">
  <p class="site-title" >Haxxor News</p>
  <nav class="site-nav">
    <div class="trigger">
      <%= link_to 'Create Post', nil, {:class => 'page-link'} %>
    </div>
  </nav>
</div>
```

Also, heads up there are two hash styles:
```ruby
# the old style
{:class => 'page-link'}
# and the newer style (introduced 11 years ago with Ruby 1.9)
{class: 'page-link'}
```

Generally it's better to use the newer style unless you have to interpolate the key.

File: app/helpers/home_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
+end
```
Comment (efatsi) [inline app/helpers/home_helper.rb]: it's good practice to remove auto-generated files that you don't need.

File: app/views/home/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+<header class="site-header" role="banner">
+  <div class="wrapper">
+      <p class="site-title" >Haxxor News</p>
+			<nav class="site-nav">
+				<div class="trigger">
+					<%= link_to 'Create Post', nil, {class: 'page-link'} %>
+				</div>
+			</nav>
+	</div>
+</header>
```
Comment (efatsi) [inline app/views/home/index.html.erb]: following up on @dbernheisel's note, being clean and consistent with spacing goes a long way towards keeping your code readable and less annoying in the future. looks like you have a handful of extra space chunks in this file

**Pull request merged**